bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/internal/processing/Tracer.kt

```diff
@@ -2,20 +2,23 @@ package com.bugsnag.android.performance.internal.processing
 
 import android.os.SystemClock
 import com.bugsnag.android.performance.Span
+import com.bugsnag.android.performance.SpanEndCallback
 import com.bugsnag.android.performance.internal.InternalDebug
 import com.bugsnag.android.performance.internal.ProbabilitySampler
 import com.bugsnag.android.performance.internal.Sampler
 import com.bugsnag.android.performance.internal.SpanImpl
 import com.bugsnag.android.performance.internal.Worker
 
-internal class Tracer : BatchingSpanProcessor() {
+internal class Tracer(spanEndCallbacks: Array<SpanEndCallback>) : BatchingSpanProcessor() {
 
     private var lastBatchSendTime = SystemClock.elapsedRealtime()
 
     internal var sampler: Sampler = ProbabilitySampler(1.0)
 
     internal var worker: Worker? = null
 
+    internal var spanEndCallbacks: MutableList<SpanEndCallback> = spanEndCallbacks.toMutableList()
```

Comment:
This should be an `Array<SpanEndCallback>` and not a `MutableList`. It should also be marked `@VisibleForTesting` and a `val` and not `var`.